src/components/checkbox/checkbox.js

```diff
@@ -96,6 +101,23 @@ class Checkbox extends mixin(createComponent, initComponentBySearch, handles) {
     }
   }
 
+  setDisabled(value) {
+    if (value === undefined || disabledTypes[value] === undefined) {
```

Comment:
Great that you added sanity check here! And good attempt to use a table, which works for limited set of accepted values in general. I’m guessing that we are accepting only boolean values? Do we want to accept `'true'` and `'false'` as strings?

Reply:
I updated the code to use a boolean value for `setDisabled`, you're right that makes more sense. I also found a bug with the indeterminate checkbox when I was testing and fixed that on line `115`.